README.md

```diff
@@ -35,6 +35,7 @@ Table of Contents
   * [SVN migration to Git](#svn-migration-to-git)
   * [Git Bash](#git-bash)
   * [Setting up Git locally](#setting-up-git-locally)
+  * [Git from the inside out](https://codewords.recurse.com/issues/two/git-from-the-inside-out)
```

Comment:
Maga, you have added this in the Table Of Contents. Add it to the Git subheading :D